### PR TITLE
Add docker cli image

### PR DESCRIPTION
**Pull request by SimonMellerin**

### CODE REVIEW

File: docker/franken-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+RUN docker-php-ext-enable sqlsrv pdo_sqlsrv
+
+# Cleanup.
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY --from=composer /usr/bin/composer /usr/bin/composer
+
+WORKDIR /var/www
+
+RUN composer require makinacorpus/db-tools-bundle $TAG
```
Comment (pounard) [inline docker/franken-image/Dockerfile:36]: I would use the syntax `makinacorpus/db-tools-bundle:'$TAG'` instead, in order to be able to add other packages, such as `db-tools-bundle/pack-fr-fr`, `symfony/symfony/password-hasher` etc... which are optional dependencies.

**Comment by SimonMellerin** [comment]
The CI for this PR won't work until it's merged on `main` (due to changes of the docker CI image name).

File: .github/workflows/build-and-publish-ci-image.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   build-and-push-image:
-    name: Push Docker image to Docker Hub
+    name: Push Docker ci image to Docker Hub
```
Comment (Lonnytunes) [inline .github/workflows/build-and-publish-ci-image.yml:17]:
```suggestion
    name: Push Docker CI image to Docker Hub
```

File: .github/workflows/build-and-publish-franken-image.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+name: Build Docker franken image
```
Comment (Lonnytunes) [inline .github/workflows/build-and-publish-franken-image.yml:1]:
```suggestion
name: Build Docker FrankenPHP image
```

**Pull request merged**